packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -22,21 +22,22 @@ const GlobalClassNames = {
 
 export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
   const { className, titleLabelClassName, theme, vertical, disabled, showTransitions, showValue } = props;
-  const { palette } = theme;
+  const { palette, semanticColors } = theme;
   const classNames = getGlobalClassNames(GlobalClassNames, theme);
 
   // Tokens
-  const sliderInteractedActiveSectionColor = palette.themePrimary;
-  const sliderInteractedInactiveSectionColor = palette.themeLight;
-  const sliderRestActiveSectionColor = palette.neutralSecondary;
-  const sliderRestInactiveSectionColor = palette.neutralTertiaryAlt;
+  const sliderInteractedActiveSectionColor = semanticColors.inputBackgroundCheckedHovered;
+  const sliderHoverSectionColor = semanticColors.inputBackgroundChecked;
```

Comment:
this is for the active section I presume? isn't hover and pressed the same, hence the interactive slot?

Reply:
@phkuo here is my proposal for renaming the "Interacted" tokens: 
`hoverActiveSectionColor` map to slot: `inputBackgroundChecked`(themePrimary) 
`pressedActiveSectionColor` map to slot:`inputBackgroundCheckedHovered`(themeDark)

I propose `hoverPressedinactiveSectionColor` for the now renamed slot: `inputPlaceholderBackgroundChecked` (themeLighter)
since it is used by both hover and pressed inactive states.

thoughts

Reply:
I like this proposal. Having interacted slots and hover slots at the same time is confusing - having just separate hover & pressed slots is more clear.

Reply:
sounds great! I would use past tense of hover though, like we do elsewhere.